### ISSUE TITLE
Resume task cmd

### ISSUE THREAD

**Issue by jtagcat** [almarklein/timetagger_cli#15]
It'd be great I could resume the last task, or pick from recent items with a TUI.

**Comment by almarklein** [comment]
You mean like this?
<img width="802" alt="image" src="https://github.com/almarklein/timetagger_cli/assets/3015475/50fbef5a-e76a-4369-8f1d-fc76b592cf8d">

**Comment by jtagcat** [comment]
Yes.

**Comment by martin-braun** [comment]
A good start would be to implement it with a `--last` arg, so we can resume the last task without much headache. A TUI would be preferred without the arg, but that can come later.

What do you think? @almarklein

**Comment by almarklein** [comment]
> Yes.

Lol, I totally missed that this issue was on the CLI repo, so I thought you had simply overlooked the feature in the app. Sorry!

> A good start would be to implement it with a `--last` arg, so we can resume the last task without much headache. A TUI would be preferred without the arg, but that can come later.

Yes, that sounds like a great start. A next step could be listing the 8 or so previous records, with numbers 1-8 before them, and allowing to enter a number for the record to resume, with the last record (which might actually be the first in the list?) be the default, so simply hitting enter would be a quick way to resume the last record.

**Comment by martin-braun** [comment]
> Yes, that sounds like a great start. A next step could be listing the 8 or so previous records, with numbers 1-8 before them, and allowing to enter a number for the record to resume, with the last record (which might actually be the first in the list?) be the default, so simply hitting enter would be a quick way to resume the last record.

I love this. I think it would be nice to have the latest task at the bottom to better reflect the timeline, so: 

```
[8] oldest task
[7] buying food
[6] ...
...
[1] most recent task

Which task do you want to resume? (1)▐
```

**Comment by kwillno** [comment]
Some of the functionality described here was implemented in #16 .  



> I love this. I think it would be nice to have the latest task at the bottom to better reflect the timeline, so:
> 
> ```
> [8] oldest task
> [7] buying food
> [6] ...
> ...
> [1] most recent task
> 
> Which task do you want to resume? (1)▐
> ```

This looks like more elegant way to do this, perhaps something more for the future?
